README.md

```diff
@@ -97,7 +97,7 @@ disk(() -> big_array(n))
 # are the inputs to the function:
 disk(big_array, n)
 ```
-Internally, by default this will call `GC.gc(false)`, which performs an incremental collection of only the "young" objects in memory.
+Internally, by default this version of the function `disk` will call `GC.gc(false)` internally, which performs an incremental collection of only the "young" objects in memory.
```

Comment:
Now the sentence says "internally" twice, maybe remove the first one?

Reply:
Good point - just fixed that